src/config/statuses.config.ts

```diff
@@ -0,0 +1,19 @@
+export const statuses = <Statuses>{
```

Comment:
Statuses can be stored for checks in constants, but will also need to be stored on DB level via codes/codetypes. The `code` property will always be immutable to avoid errors when checking for a specific `status-code` property.